patches/tModLoader/Terraria/Social/Steam/CoreSocialModule.TML.cs

```diff
@@ -0,0 +1,25 @@
+using System.IO;
+
+namespace Terraria.Social.Steam
+{
+	public partial class CoreSocialModule : ISocialModule
+	{
+		public const string WindowsSteamCloudPath = "C:\\Program Files (x86)\\Steam\\userdata";
+		public const string LinuxSteamCloudPath = "~/.local/share/Steam/userdata";
+		public const string MacSteamCloudPath = "~/Library/Application Support/Steam/userdata";
```

Comment:
Is this really necessary? I can easily imagine people with different drive letters or install locations. None of the other steam cloud code uses them.

Reply:
This code doesn't even use it. 
I was thinking might be nice to have for supportability, but it's not like you can actually move the files around yourself - only copy them away from there to local manually.
Not really useful at this juncture, will remove.